cmake/pcl_targets.cmake

```diff
@@ -385,6 +385,21 @@ macro(PCL_ADD_TEST _name _exename)
   # must link explicitly against boost only on Windows
   target_link_libraries(${_exename} ${Boost_LIBRARIES})
 
+  #Only applies to MSVC
+  if(MSVC)    
+    #Requires CMAKE version 3.13.0
+    if(${CMAKE_VERSION} VERSION_LESS "3.13.0" AND (NOT ArgumentWarningShown))
+      message(WARNING "Arguments to unit projects is not added - requires cmake 3.13")
```

Comment:
The warning tells what *did not* happen. But is this a severe problem? Is something broken now? What to do about this?

I think either the warning should explain this, or not appear at all.

Reply:
The unit tests which has arguments to PCD files etc., won't have them in the project settings in VS.
So launching/debugging them will exit with various errors of clouds not found or sizes/feature sizes of 0.

You can manually add the arguments.

Reply:
I'll update the warning that the user has to manually add them to run them from VS.